static/css/style.css

```diff
@@ -2,10 +2,10 @@
     /* P = Primary | S = Secondary | T = Tertiary */
     --p-orange: #E84610;
     --p-blue: #009FE3;
-    --p-grey: #4A4A4F;
+    --p-grey: #4a4a4f;
```

Comment:
Why are you changing this to lower case? Also, if you wanted to do that, why not for all colors? Let's keep it consistent whichever way.

Reply:
I physically didn't do this? I didn't add to, or change any of the root variables as I had no need to. But I've resolved it now.